src/views/Login.jsx

```diff
@@ -30,12 +30,6 @@ const Login = () => {
         }
     };
 
-    useEffect(() => {
-        if (token) {
-            navigate('/login'); // Redirige a otra ruta cuando el token no es null
-        }
-    }, [token, navigate]);
```

Comment:
this is the most important thing to highlight in your solution. Here , this **shouldn't** be an `useEffect()`, remember, 

### 🕷️ _With great power, comes great responsibility_

![Screenshot 2024-09-18 at 18 11 56](https://github.com/user-attachments/assets/b0a60a3b-939a-4a45-9f37-279ed2ee79f3)